protocol/avdl/chat1/common.avdl

```diff
@@ -187,6 +187,12 @@ protocol common {
     bytes k; // Verifying key
   }
 
+  record SignEncryptedData {
+    int   v; // version = 1
+    bytes b; // signEncryptedData (output of signencrypt.SealWhole)
+    bytes n; // nonce
```

Comment:
Ok actually I'm wrong about that. We do sometimes ask "who wrote this" in the sense that in a conversation with 2 messages it's meaningful to ask "who wrote the _first one_ and who wrote the _second one_". The messages have kind of an identity of their own, with respect to the linear history of messages in this thread. So if `foo` could come along and claim ownership of message 1, when they didn't actually write it, that's different from just signing their own copy of the same plaintext, and probably counts as an attack. This is super duper corner case even by my standards, but I'll think about it more.